packages/synapse-react-client/src/components/TanStackTable/StyledTanStackTable.tsx

```diff
@@ -89,6 +94,7 @@ export default function StyledTanStackTable<
         {...tableSlotProps}
         style={{
           ...columnSizeVars,
+          tableLayout: 'fixed',
```

Comment:
Fixes the re-sizing issue. This seems to work well for both dynamic-width tables and fullWidth (100%) tables